networking/configuring-ipfailover.adoc

```diff
@@ -8,17 +8,24 @@ toc::[]
 
 This topic describes configuring IP failover for pods and services on your {product-title} cluster.
 
```

Comment:
"Configuring IP failover" is used twice in this section. Once at the top: https://74064--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/configuring-ipfailover.html#nw-ipfailover-configuration_configuring-ipfailover and once again after the environment variables https://74064--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/configuring-ipfailover.html#nw-ipfailover-configuration_configuring-ipfailover 

Like this in the live docs: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html/networking/configuring-ipfailover#doc-wrapper

Wondering if you'd be open to re-titling one or the other?

Reply:
Renamed the child module. Thanks!